superfly/flyctl
# Adding flaps code to launch machine

This PR swaps launch machine code with flaps, it's currently branched off the machine refactor branch since it uses the new machine run command

To test, run:
`go run . machine run gcr.io/kuar-demo/kuard-amd64:purple --app testingtestingtesting -r dev`
You should see the following output:
![Screen_Shot_2022-03-28_at_4_07_09_PM](https://user-images.githubusercontent.com/3229484/160501927-97001b98-9d68-4c84-acfb-6c4b90698531.png)
